omniauth-oauth2.gemspec

```diff
@@ -4,7 +4,7 @@ require "omniauth-oauth2/version"
 
 Gem::Specification.new do |gem|
   gem.add_dependency "oauth2",     [">= 1.4", "< 3"]
-  gem.add_dependency "omniauth",   "~> 2.0"
+  gem.add_dependency "omniauth",   "~> 2.0.2"
```

Comment:
Because the errors we are rescuing are [introduced in version 2.0.2 of omniauth](https://gitlab.com/oauth-xx/oauth2/-/blob/866dc365bfe0d64f6cc56295a38ccd5b24143836/CHANGELOG.md#L74) this seemed like a good update.

Reply:
rejecting 2.1+?

Reply:
@nov Good point, this is somewhat inflexible, isn't it? Maybe we should change to:

```ruby
 gem.add_dependency "omniauth",   "~> 2.0.2, "~> 2.1"
```

Not sure if this is valid syntax for a gemspec but could give it a try

Reply:
Specify like we do oauth2 above

Reply:
and I realized you should specify v2.0.2 of oauth2 gem, not omniauth.